apps/frontend/vite.config.mts

```diff
@@ -0,0 +1,86 @@
+/// <reference types="vitest" />
+import { resolve } from 'path'
+import { defineConfig, normalizePath } from 'vite'
+
+import react from '@vitejs/plugin-react'
+// viteStaticCopy contains some `require`, so we need to have our config as .mts instead of .ts.
+// https://vitejs.dev/guide/troubleshooting.html#this-package-is-esm-only
+import { viteStaticCopy } from 'vite-plugin-static-copy'
+import viteTsConfigPaths from 'vite-tsconfig-paths'
+
+export default defineConfig({
+  base: '',
+  cacheDir: '../../node_modules/.vite/frontend',
+
+  server: {
+    port: 4200,
+    host: 'localhost',
+    // Allows workers to fetch the needed .ts file
+    fs: {
+      allow: ['../../'],
+    },
+  },
+
+  preview: {
+    port: 4300,
+    host: 'localhost',
+  },
+
+  plugins: [
+    viteTsConfigPaths({
+      root: '../../',
+    }),
+    // Enables proper hot module reloading
+    react({
+      include: '**/*.tsx',
+    }),
+    // Nx executor for vite does not support `assets` prop for copying files.
```

Comment:
can you link to an nx issue here so we can track this support in the future?

Reply:
There is none, not sure if it is an oversight or on purpose. I can create one